src/Aspire.Hosting/Utils/FileUtil.cs

```diff
@@ -7,7 +7,7 @@ internal static class FileUtil
 {
     public static string FindFullPathFromPath(string command)
     {
-        ArgumentOutOfRangeException.ThrowIfNullOrWhiteSpace(command, nameof(command));
+        ArgumentException.ThrowIfNullOrWhiteSpace(command);
```

Comment:
This is an internal API that is only called with constant strings. We should just turn this into a `Debug.Assert`.